src/js/components/Brick.js

```diff
@@ -0,0 +1,47 @@
+// (C) Copyright 2014-2015 Hewlett Packard Enterprise Development LP
+
+import React, { Component } from 'react';
+import Anchor from './Anchor';
+
+const CLASS_ROOT = "brick";
```

Comment:
@alansouzati: Does this commit (https://github.com/grommet/grommet/commit/3859237a45f85a83327c85b55fe74b0766b822b7) fix what you are referring to here?